<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>680</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>680</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QMainWindow {
	background-color:#f0f0f0;
}
QCheckBox {
	padding:2px;
}
QCheckBox:hover {
	border-radius:4px;
	border-style:solid;
	border-width:1px;
	padding-left: 1px;
	padding-right: 1px;
	padding-bottom: 1px;
	padding-top: 1px;
	border-color: rgb(255,150,60);
	background-color:qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(190, 90, 50, 50), stop:1 rgba(250, 130, 40, 50));
}
QCheckBox::indicator:checked {
	border-radius:4px;
	border-style:solid;
	border-width:1px;
	border-color: rgb(246, 134, 86);
  	background-color:rgb(246, 134, 86)
}
QCheckBox::indicator:unchecked {
	border-radius:4px;
	border-style:solid;
	border-width:1px;
	border-color:rgb(246, 134, 86);
  	background-color:rgb(255,255,255);
}
QColorDialog {
	background-color:#f0f0f0;
}
QComboBox {
	color:rgb(81,72,65);
	background: #ffffff;
}
QComboBox:editable {
	background: #ffffff;
	color: rgb(81,72,65);
	selection-color:rgb(81,72,65);
	selection-background-color: #ffffff;
}
QComboBox QAbstractItemView {
	color:rgb(81,72,65);	
	background: #ffffff;
	selection-color: #ffffff;
	selection-background-color: rgb(246, 134, 86);
}
QComboBox:!editable:on, QComboBox::drop-down:editable:on {
	color:  #1e1d23;	
	background: #ffffff;
}
QDateTimeEdit {
	color:rgb(81,72,65);
	background-color: #ffffff;
}
QDateEdit {
	color:rgb(81,72,65);
	background-color: #ffffff;
}
QDialog {
	background-color:#f0f0f0;
}
QDoubleSpinBox {
	color:rgb(81,72,65);
	background-color: #ffffff;
}
QFontComboBox {
	color:rgb(81,72,65);
	background-color: #ffffff;
}
QLabel {
	color:rgb(17,17,17);
}
QLineEdit {
	background-color:rgb(255,255,255);
	selection-background-color:rgb(236,116,64);
	color:rgb(17,17,17);
}
QMenuBar {
	color:rgb(223,219,210);
	background-color:rgb(65,64,59);
}
QMenuBar::item {
	padding-top:4px;
	padding-left:4px;
	padding-right:4px;
	color:rgb(223,219,210);
	background-color:rgb(65,64,59);
}
QMenuBar::item:selected {
	color:rgb(255,255,255);
	padding-top:2px;
	padding-left:2px;
	padding-right:2px;
	border-top-width:2px;
	border-left-width:2px;
	border-right-width:2px;
	border-top-right-radius:4px;
	border-top-left-radius:4px;
	border-style:solid;
	background-color:rgb(65,64,59);
	border-top-color: rgb(47,47,44);
	border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(90, 87, 78, 255), stop:1 rgba(47,47,44, 255));
	border-left-color:  qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgba(90, 87, 78, 255), stop:1 rgba(47,47,44, 255));
}
QMenu {
	color:rgb(223,219,210);
	background-color:rgb(65,64,59);
}
QMenu::item {
	color:rgb(223,219,210);
	padding-left:20px;
	padding-top:4px;
	padding-bottom:4px;
	padding-right:10px;
}
QMenu::item:selected {
	color:rgb(255,255,255);
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(225, 108, 54, 255), stop:1 rgba(246, 134, 86, 255));
	border-style:solid;
	border-width:3px;
	padding-left:17px;
	padding-top:4px;
	padding-bottom:4px;
	padding-right:7px;
	border-bottom-color:qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(175,85,48,255), stop:1 rgba(236,114,67, 255));
	border-top-color:qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(253,156,113,255), stop:1 rgba(205,90,46, 255));
	border-right-color:qlineargradient(spread:pad, x1:0, y1:0.5, x2:1, y2:0.5, stop:0 rgba(253,156,113,255), stop:1 rgba(205,90,46, 255));
	border-left-color:qlineargradient(spread:pad, x1:1, y1:0.5, x2:0, y2:0.5, stop:0 rgba(253,156,113,255), stop:1 rgba(205,90,46, 255));
}
QPlainTextEdit {
	border-width: 1px;
	border-style: solid;
	border-color:transparent;
	color:rgb(17,17,17);
	selection-background-color:rgb(236,116,64);
}
QProgressBar {
	text-align: center;
	color: rgb(0, 0, 0);
	border-width: 1px; 
	border-radius: 10px;
	border-style: inset;
	border-color: rgb(150,150,150);
	background-color:rgb(221,221,219);
}
QProgressBar::chunk:horizontal {
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(225, 108, 54, 255), stop:1 rgba(246, 134, 86, 255));
	border-style: solid;
	border-radius:8px;
	border-width:1px;
	border-bottom-color:qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(175,85,48,255), stop:1 rgba(236,114,67, 255));
	border-top-color:qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(253,156,113,255), stop:1 rgba(205,90,46, 255));
	border-right-color:qlineargradient(spread:pad, x1:0, y1:0.5, x2:1, y2:0.5, stop:0 rgba(253,156,113,255), stop:1 rgba(205,90,46, 255));
	border-left-color:qlineargradient(spread:pad, x1:1, y1:0.5, x2:0, y2:0.5, stop:0 rgba(253,156,113,255), stop:1 rgba(205,90,46, 255));
}
QPushButton{
	color:rgb(17,17,17);
	border-width: 1px;
	border-radius: 6px;
	border-bottom-color: rgb(150,150,150);
	border-right-color: rgb(165,165,165);
	border-left-color: rgb(165,165,165);
	border-top-color: rgb(180,180,180);
	border-style: solid;
	padding: 4px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:hover{
	color:rgb(17,17,17);
	border-width: 1px;
	border-radius:6px;
	border-top-color: rgb(255,150,60);
	border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(200, 70, 20, 255), stop:1 rgba(255,150,60, 255));
	border-left-color:  qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgba(200, 70, 20, 255), stop:1 rgba(255,150,60, 255));
	border-bottom-color: rgb(200,70,20);
	border-style: solid;
	padding: 2px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:default{
	color:rgb(17,17,17);
	border-width: 1px;
	border-radius:6px;
	border-top-color: rgb(255,150,60);
	border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(200, 70, 20, 255), stop:1 rgba(255,150,60, 255));
	border-left-color:  qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgba(200, 70, 20, 255), stop:1 rgba(255,150,60, 255));
	border-bottom-color: rgb(200,70,20);
	border-style: solid;
	padding: 2px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:pressed{
	color:rgb(17,17,17);
	border-width: 1px;
	border-radius: 6px;
	border-width: 1px;
	border-top-color: rgba(255,150,60,200);
	border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(200, 70, 20, 255), stop:1 rgba(255,150,60, 200));
	border-left-color:  qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgba(200, 70, 20, 255), stop:1 rgba(255,150,60, 200));
	border-bottom-color: rgba(200,70,20,200);
	border-style: solid;
	padding: 2px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:disabled{
	color:rgb(174,167,159);
	border-width: 1px;
	border-radius: 6px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(200, 200, 200, 255), stop:1 rgba(230, 230, 230, 255));
}
QRadioButton {
	padding: 1px;
}
QRadioButton::indicator:checked {
	height: 10px;
	width: 10px;
	border-style:solid;
	border-radius:5px;
	border-width: 1px;
	border-color: rgba(246, 134, 86, 255);
	color: #a9b7c6;
	background-color:rgba(246, 134, 86, 255);
}
QRadioButton::indicator:!checked {
	height: 10px;
	width: 10px;
	border-style:solid;
	border-radius:5px;
	border-width: 1px;
	border-color: rgb(246, 134, 86);
	color: #a9b7c6;
	background-color: transparent;
}
QScrollArea {
	color: #FFFFFF;
	background-color:#f0f0f0;
}
QSlider::groove {
	border-style: solid;
	border-width: 1px;
	border-color: rgb(207,207,207);
}
QSlider::groove:horizontal {
	height: 5px;
	background: rgb(246, 134, 86);
}
QSlider::groove:vertical {
	width: 5px;
	background: rgb(246, 134, 86);
}
QSlider::handle:horizontal {
	background: rgb(253,253,253);
	border-style: solid;
	border-width: 1px;
	border-color: rgb(207,207,207);
	width: 12px;
	margin: -5px 0;
	border-radius: 7px;
}
QSlider::handle:vertical {
	background: rgb(253,253,253);
	border-style: solid;
	border-width: 1px;
	border-color: rgb(207,207,207);
	height: 12px;
	margin: 0 -5px;
	border-radius: 7px;
}
QSlider::add-page:horizontal {
 	background: white;
}
QSlider::add-page:vertical {
	background: white;
}
QSlider::sub-page:horizontal {
	background: rgb(246, 134, 86);
}
QSlider::sub-page:vertical {
  	background: rgb(246, 134, 86);
}
QStatusBar {
	color:rgb(81,72,65);
}
QSpinBox {
	color:rgb(81,72,65);
	background-color: #ffffff;
}
QScrollBar:horizontal {
	max-height: 20px;
	border: 1px transparent grey;
	margin: 0px 20px 0px 20px;
}
QScrollBar::handle:horizontal {
	background: rgb(253,253,253);
	border-style: solid;
	border-width: 1px;
	border-color: rgb(207,207,207);
	border-radius: 7px;
	min-width: 25px;
}
QScrollBar::handle:horizontal:hover {
	background: rgb(253,253,253);
	border-style: solid;
	border-width: 1px;
	border-color: rgb(255,150,60);
	border-radius: 7px;
	min-width: 25px;
}
QScrollBar::add-line:horizontal {
  	border: 1px solid;
  	border-color: rgb(207,207,207);
  	border-top-right-radius: 7px;
  	border-top-left-radius: 7px;
  	border-bottom-right-radius: 7px;
  	background: rgb(255, 255, 255);
  	width: 20px;
  	subcontrol-position: right;
  	subcontrol-origin: margin;
}
QScrollBar::add-line:horizontal:hover {
  	border: 1px solid;
  	border-top-right-radius: 7px;
  	border-top-left-radius: 7px;
  	border-bottom-right-radius: 7px;
  	border-color: rgb(255,150,60);
  	background: rgb(255, 255, 255);
  	width: 20px;
  	subcontrol-position: right;
  	subcontrol-origin: margin;
}
QScrollBar::add-line:horizontal:pressed {
  	border: 1px solid grey;
  	border-top-left-radius: 7px;
  	border-top-right-radius: 7px;
  	border-bottom-right-radius: 7px;
  	background: rgb(231,231,231);
  	width: 20px;
  	subcontrol-position: right;
  	subcontrol-origin: margin;
}
QScrollBar::sub-line:horizontal {
  	border: 1px solid;
  	border-color: rgb(207,207,207);
  	border-top-right-radius: 7px;
  	border-top-left-radius: 7px;
  	border-bottom-left-radius: 7px;
  	background: rgb(255, 255, 255);
  	width: 20px;
  	subcontrol-position: left;
  	subcontrol-origin: margin;
}
QScrollBar::sub-line:horizontal:hover {
  	border: 1px solid;
  	border-color: rgb(255,150,60);
  	border-top-right-radius: 7px;
  	border-top-left-radius: 7px;
  	border-bottom-left-radius: 7px;
  	background: rgb(255, 255, 255);
  	width: 20px;
  	subcontrol-position: left;
  	subcontrol-origin: margin;
}
QScrollBar::sub-line:horizontal:pressed {
  	border: 1px solid grey;
  	border-top-right-radius: 7px;
  	border-top-left-radius: 7px;
  	border-bottom-left-radius: 7px;
  	background: rgb(231,231,231);
  	width: 20px;
  	subcontrol-position: left;
  	subcontrol-origin: margin;
}
QScrollBar::left-arrow:horizontal {
  	border: 1px transparent grey;
  	border-top-left-radius: 3px;
  	border-bottom-left-radius: 3px;
  	width: 6px;
  	height: 6px;
  	background: rgb(230,230,230);
}
QScrollBar::right-arrow:horizontal {
	border: 1px transparent grey;
	border-top-right-radius: 3px;
	border-bottom-right-radius: 3px;
  	width: 6px;
  	height: 6px;
 	background: rgb(230,230,230);
}
QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal {
 	background: none;
} 
QScrollBar:vertical {
	max-width: 20px;
	border: 1px transparent grey;
	margin: 20px 0px 20px 0px;
}
QScrollBar::add-line:vertical {
	border: 1px solid;
	border-color: rgb(207,207,207);
	border-bottom-right-radius: 7px;
	border-bottom-left-radius: 7px;
	border-top-left-radius: 7px;
	background: rgb(255, 255, 255);
  	height: 20px;
  	subcontrol-position: bottom;
  	subcontrol-origin: margin;
}
QScrollBar::add-line:vertical:hover {
  	border: 1px solid;
  	border-color: rgb(255,150,60);
  	border-bottom-right-radius: 7px;
  	border-bottom-left-radius: 7px;
  	border-top-left-radius: 7px;
  	background: rgb(255, 255, 255);
  	height: 20px;
  	subcontrol-position: bottom;
  	subcontrol-origin: margin;
}
QScrollBar::add-line:vertical:pressed {
  	border: 1px solid grey;
  	border-bottom-left-radius: 7px;
  	border-bottom-right-radius: 7px;
  	border-top-left-radius: 7px;
  	background: rgb(231,231,231);
  	height: 20px;
  	subcontrol-position: bottom;
  	subcontrol-origin: margin;
}
QScrollBar::sub-line:vertical {
  	border: 1px solid;
  	border-color: rgb(207,207,207);
  	border-top-right-radius: 7px;
  	border-top-left-radius: 7px;
  	border-bottom-left-radius: 7px;
  	background: rgb(255, 255, 255);
  	height: 20px;
  	subcontrol-position: top;
  	subcontrol-origin: margin;
}
QScrollBar::sub-line:vertical:hover {
  	border: 1px solid;
  	border-color: rgb(255,150,60);
  	border-top-right-radius: 7px;
  	border-top-left-radius: 7px;
  	border-bottom-left-radius: 7px;
	background: rgb(255, 255, 255);
  	height: 20px;
  	subcontrol-position: top;
  	subcontrol-origin: margin;
}
QScrollBar::sub-line:vertical:pressed {
  	border: 1px solid grey;
  	border-top-left-radius: 7px;
  	border-top-right-radius: 7px;
  	background: rgb(231,231,231);
 	height: 20px;
  	subcontrol-position: top;
  	subcontrol-origin: margin;
}
QScrollBar::handle:vertical {
	background: rgb(253,253,253);
	border-style: solid;
	border-width: 1px;
	border-color: rgb(207,207,207);
	border-radius: 7px;
	min-height: 25px;
}
QScrollBar::handle:vertical:hover {
	background: rgb(253,253,253);
	border-style: solid;
	border-width: 1px;
	border-color: rgb(255,150,60);
	border-radius: 7px;
	min-height: 25px;
}
QScrollBar::up-arrow:vertical {
	border: 1px transparent grey;
  	border-top-left-radius: 3px;
	border-top-right-radius: 3px;
  	width: 6px;
  	height: 6px;
  	background: rgb(230,230,230);
}
QScrollBar::down-arrow:vertical {
  	border: 1px transparent grey;
  	border-bottom-left-radius: 3px;
  	border-bottom-right-radius: 3px;
  	width: 6px;
  	height: 6px;
  	background: rgb(230,230,230);
}
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
  	background: none;
}
QTabWidget {
	color:rgb(0,0,0);
	background-color:rgb(247,246,246);
}
QTabWidget::pane {
	border-color: rgb(180,180,180);
	background-color:rgb(247,246,246);
	border-style: solid;
	border-width: 1px;
  	border-radius: 6px;
}
QTabBar::tab {
	padding-left:4px;
	padding-right:4px;
	padding-bottom:2px;
	padding-top:2px;
	color:rgb(81,72,65);
  	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(221,218,217,255), stop:1 rgba(240,239,238,255));
	border-style: solid;
	border-width: 1px;
  	border-top-right-radius:4px;
	border-top-left-radius:4px;
	border-top-color: rgb(180,180,180);
	border-left-color: rgb(180,180,180);
	border-right-color: rgb(180,180,180);
	border-bottom-color: transparent;
}
QTabBar::tab:selected, QTabBar::tab:last:selected, QTabBar::tab:hover {
  	background-color:rgb(247,246,246);
  	margin-left: 0px;
  	margin-right: 1px;
}
QTabBar::tab:!selected {
	margin-top: 1px;
	margin-right: 1px;
}
QTextEdit {
	border-width: 1px;
	border-style: solid;
	border-color:transparent;
	color:rgb(17,17,17);
	selection-background-color:rgb(236,116,64);
}
QTimeEdit {
	color:rgb(81,72,65);
	background-color: #ffffff;
}
QToolBox {
	color:rgb(81,72,65);
	background-color: #ffffff;
}
QToolBox::tab {
	color:rgb(81,72,65);
	background-color: #ffffff;
}
QToolBox::tab:selected {
	color:rgb(81,72,65);
	background-color: #ffffff;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>页</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QDateTimeEdit" name="startTimeEdit">
           <property name="contextMenuPolicy">
            <enum>Qt::DefaultContextMenu</enum>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::UpDownArrows</enum>
           </property>
           <property name="dateTime">
            <datetime>
             <hour>0</hour>
             <minute>0</minute>
             <second>0</second>
             <year>2020</year>
             <month>10</month>
             <day>10</day>
            </datetime>
           </property>
           <property name="date">
            <date>
             <year>2020</year>
             <month>10</month>
             <day>10</day>
            </date>
           </property>
           <property name="maximumDateTime">
            <datetime>
             <hour>23</hour>
             <minute>59</minute>
             <second>59</second>
             <year>2030</year>
             <month>11</month>
             <day>6</day>
            </datetime>
           </property>
           <property name="minimumDateTime">
            <datetime>
             <hour>0</hour>
             <minute>0</minute>
             <second>0</second>
             <year>2020</year>
             <month>4</month>
             <day>20</day>
            </datetime>
           </property>
           <property name="maximumDate">
            <date>
             <year>2030</year>
             <month>11</month>
             <day>6</day>
            </date>
           </property>
           <property name="currentSection">
            <enum>QDateTimeEdit::MonthSection</enum>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>To</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateTimeEdit" name="endTimeEdit">
           <property name="currentSection">
            <enum>QDateTimeEdit::MonthSection</enum>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tableView"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="tabelLabel">
           <property name="text">
            <string>xlabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="exportBtn">
           <property name="text">
            <string>export</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="updateTableBtn">
           <property name="text">
            <string>update</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,0">
       <item>
        <widget class="QWidget" name="virualWgt" native="true"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="plotLabel">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QTextEdit" name="radioEdit"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QComboBox" name="portBox"/>
           </item>
           <item>
            <widget class="QPushButton" name="cornectBtn">
             <property name="text">
              <string>connect</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="refreshBtn">
             <property name="text">
              <string>refresh</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="breakBtn">
             <property name="text">
              <string>disconnect</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="clearBtn">
             <property name="text">
              <string>clean up</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="InsertCheckBox">
             <property name="text">
              <string>Insert Database</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
